src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import React from 'react';
+import { FilterParam } from '../../types/FilterParam';
+import { Todo } from '../../types/Todo';
+import {
+  checkCompletedTodo,
+  counterOfActiveTodos,
+} from '../../Helpers/helpers';
+
+type Props = {
+  todos: Todo[],
+  filterType: FilterParam,
+  setFilterType: (type: FilterParam) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterType,
+  setFilterType,
+}) => {
+  const changedFilter = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    const href = event.currentTarget.text;
+
+    switch (href) {
+      case 'All':
```

Comment:
```suggestion
      case FilterParam.All:
```
Don't hardcore the value. Use enum instead